### ISSUE TITLE
Update gtest

### ISSUE THREAD

**Issue by hobu** [PDAL/PDAL#2564]
Our gtest is old and we run into https://github.com/google/googletest/issues/1373 when we compile on MSVC 2017

**Comment by abellgithub** [comment]
What is this issue?  I can't figure it out from the ticket.

**Comment by hobu** [comment]
Both PDAL and Entwine use gtest 1.8.0

```
D:\a\1\s\test\gtest-1.8.0\include\gtest/internal/gtest-port.h(996): error C2220: warning treated as error - no 'object' file generated
D:\a\1\s\test\gtest-1.8.0\include\gtest/internal/gtest-port.h(996): warning C4996: 'std::tr1': warning STL4002: The non-Standard std::tr1 namespace and TR1-only machinery are deprecated and will be REMOVED. You can define _SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING to acknowledge that you have received this warning.
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\tuple(1333): note: see declaration of 'std::tr1'
D:
```

**Comment by abellgithub** [comment]
It looks like we're at 1.7, not 1.8.  The latest is 1.8.1 AFAICT.

**Comment by abellgithub** [comment]
Fixed with 47bd4eeecb71ea877b029ba296088b0b506d876e
